man/toTinytable.Rd

```diff
@@ -0,0 +1,46 @@
+\name{toTinytable}
+\alias{toTinytable}
+\title{
+Convert \code{tabular} object to \code{tinytable} format.
+}
+\description{
+Converts the output of the \code{\link{tabular}} and related
+functions to a format consistent with the output of the
+\code{\link[tinytable]{tt}} function, so that it can be 
+customized using the \pkg{tinytable} package.
+}
+\usage{
+toTinytable(table, ...)
+}
+\arguments{
+  \item{table}{
+An object of class \code{tabular}.
+}
+  \item{...}{
+Additional arguments to pass to \code{\link[tinytable]{tt}}.
+}
+}
+\value{
+An object of class \code{tinytable}, suitable for passing
+to functions in the \pkg{tinytable} package. These tables
+can be exported to several formats, including LaTeX, HTML,
+Markdown, Word, Typst, PDF, and PNG.
+}
+
+\seealso{
+\code{\link[tinytable]{tinytable-package}} 
+}
+\examples{
+if (requireNamespace("tinytable") && 
+    (!requireNamespace("pkgdown") || !pkgdown::in_pkgdown())) {
```

Comment:
Is there a reason this wouldn't work in `pkgdown`?

Reply:
I copied this from `man/toKable.R` and figured you had a good reason to keep this there. But I don't use `pkgdown` so I don't know. Should I remove that? https://github.com/dmurdoch/tables/blob/9ef7a0ede239b0480797a0ce540d4ef12d408397/man/toKable.Rd#L46

Reply:
https://github.com/dmurdoch/tables/pull/20/commits/61a27982e462ff640a550d550097bc90e12ee69a removes the `pkgdown` check in the example section of `toTinytable.Rd`. Unfortunately, I can't figure out how to get the table to print as HTML in the reference page of the `pkgdown` website. (Vignette works well.) 

I can force print to HTML with `print(tab,"html")`, but this only displays the HTML code, not the rendered table. I don't know if there's a way for `pkgdown` to treat the output of an `.Rd` example as raw HTML. 

In a vignette or standalone file, `knitr` handles this gracefully by calling `knit_print()`. This explains why the table I added to your vignette displays nicely.